src/thread_local.rs

```diff
@@ -49,6 +49,9 @@ impl Rng for ThreadRng {
     }
 }
 
+// FIXME: use an inner RNG which is really a CryptoRng!
+impl CryptoRng for ThreadRng {}
```

Comment:
Can you use something similar to `+impl<R: CryptoRng+?Sized> CryptoRng for Box<R> {}` here and for `ReseedingRng`?

Reply:
Huh? I already added that impl.

Reply:
I was thinking of:
```rust
impl<R: CryptoRng, Rsdr: Reseeder<R>> CryptoRng for ReseedingRng<R, Rsdr> {}
```
For a moment I thought in `ThreadRng` it could depend on the wrapped RNG too, but that was probably wrong.